src/App.js

```diff
@@ -1,25 +1,252 @@
+
+// Assets
+import "./sass/App.scss";
 
 function App() {
+  useEffect(() => {
+    document.title = "Safe Travels";
+  }, []);
+
+  // const Filter = require("bad-words");
+  // const langFilter = new Filter();
+
+  const [trips, setTrips] = useState([]);
+  const [isChecked, setIsChecked] = useState(false);
+  const [allValues, setAllValues] = useState({
+    name: "",
+    pronouns: "",
+    age: "",
+    gender: "",
+    sexuality: "",
+    ethnicity: "",
+    city: "",
+    country: "",
+    arrivalDate: "",
+    returnDate: "",
+    arrivalDate: "",
+    partnered: "",
+    safetyRating: 0,
+    faves: "",
+    advice: "",
+  });
+
+  const handleInputChange = (e) => {
+    setAllValues({
+      ...allValues,
+      [e.target.name]: e.target.value,
+    });
+
+    if (e.target.type === "checkbox") {
+      setIsChecked(!isChecked);
+      setAllValues({
+        ...allValues,
+        [e.target.name]: !isChecked,
+      });
+    }
+  };
+
+  const handleSubmit = () => {
+    const database = getDatabase(firebase);
+    const postListRef = ref(database, "posts");
+    const newPostRef = push(postListRef);
+    set(newPostRef, allValues);
+    setAllValues({
+      name: allValues.name,
+      pronouns: allValues.pronouns,
+      age: allValues.age,
+      gender: allValues.gender,
+      sexuality: allValues.sexuality,
+      ethnicity: allValues.ethnicity,
+      city: "",
+      country: "",
+      arrivalDate: "",
+      returnDate: "",
+      partnered: "",
+      safetyRating: 0,
+      faves: "",
+      advice: "",
+    });
+    setIsChecked(false);
+  };
+
+  useEffect(() => {
+    const database = getDatabase(firebase);
+    const postListRef = ref(database, "posts");
+    onValue(postListRef, (response) => {
+      const data = response.val();
+      const newState = [];
+      for (let key in data) {
+        newState.push({ key: key, tripInfo: data[key] });
+      }
+      setTrips(newState);
+    });
+  }, []);
+
+  const [countryData, setCountryData] = useState([]);
+
+  useEffect(() => {
+    fetch("https://countriesnow.space/api/v0.1/countries")
+      .then((response) => {
+        return response.json();
+      })
+      .then((data) => {
+        setCountryData(data.data);
+      });
+  }, []);
+
+  const [bgQuery, setBgQuery] = useState("beautiful");
+  const [background, setBackground] = useState("");
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const url = new URL("https://api.unsplash.com/search/photos");
+      url.search = new URLSearchParams({
+        client_id: "Gja01wZ3k0nuZjpWdToRUKRqgClg_m-jRxFHQ2hAEYY",
+        query: `${bgQuery} landscape`,
+      });
+
+      try {
+        const data = await fetch(url);
+        const response = await data.json();
+
+        const bestPic = response.results.reduce((prev, cur) => {
+          return prev.likes > cur.likes ? prev : cur;
+        });
+
+        setBackground(bestPic.urls.full);
+      } catch (error) {
+        console.log(error);
```

Comment:
gotta clean up any left over console logs :)

Reply:
took out the console.log and put in an alert for error handling!